core/storage/user/gae_models.py

```diff
@@ -1869,6 +1869,12 @@ class UserQueryModel(base_models.BaseModel):
     edited_at_least_n_exps = datastore_services.IntegerProperty(default=None)
     # Query option to check if user has edited fewer than n explorations.
     edited_fewer_than_n_exps = datastore_services.IntegerProperty(default=None)
+    # Query option to check if user has created collection.
+    created_collection = datastore_services.BooleanProperty(default=False)
+    # Query option to check if user has used LogicProof interaction in any
+    # of the explorations that they created.
+    used_logic_proof_interaction = datastore_services.BooleanProperty(
+        default=False)
```

Comment:
Given that neither of these features are ones we actively encourage people to use, can you provide a bit more context on why this is needed? Unless I missed something, the opening PR comment didn't include any issue references or specific context on why the change is needed.

Reply:
Now that we decided to remove these features from Oppia, we have to send mail to the users who have used them, just to notify them about this. This can also be used as an audit job to check the number of users. So we had chosen to add these two predicates.

Reply:
Thanks for adding this clarity--can you also mention this in the PR's top-level comment to help clarify the context behind this PR for future people who come across it?

Reply:
Done.